app/_data/docs_nav_konnect.yml

```diff
@@ -62,6 +62,8 @@
           url: /gateway-manager/serverless-gateways/provision-serverless-gateway/
         - text: Securing Backend Traffic
           url: /gateway-manager/serverless-gateways/securing-backend-traffic/
+        - text: Custom Domains
+          url: /gateway-manager/serverless-gateways/custom-dns/
```

Comment:
Please also remove the original nav entry for custom domains from the reference section, and add ones for Dev Portal and Dedicated Cloud Gateways.